server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/TransportGetSnapshotsAction.java

```diff
@@ -66,6 +67,8 @@
  */
 public class TransportGetSnapshotsAction extends TransportMasterNodeAction<GetSnapshotsRequest, GetSnapshotsResponse> {
 
+    public static final String NO_POLICY_PATTERN = "_none";
```

Comment:
nit: I think this belongs on the request more than the transport action. It is part of the interface of the action.